ddl/backfilling_dispatcher.go

```diff
@@ -493,6 +497,25 @@ func getSummaryFromLastStep(
 	return minKey, maxKey, totalKVSize, allDataFiles, allStatFiles, nil
 }
 
+func cleanupCloudStorageFiles(ctx context.Context, gTaskMeta *BackfillGlobalMeta) {
+	backend, err := storage.ParseBackend(gTaskMeta.CloudStorageURI, nil)
+	if err != nil {
+		logutil.Logger(ctx).Warn("failed to parse cloud storage uri", zap.Error(err))
```

Comment:
is jobID already included in logger in ctx? If not I think we better log more details so manually deletion will be easier

Reply:
No, but I think we can attach the jobID in next PR?